src/HUDL/HUDL.cpp

```diff
@@ -47,15 +35,42 @@ uint16_t HUDL::getObjectDictionarySize() const {
     return OBJECT_DICTIONARY_SIZE;
 }
 
-void HUDL::updateLCD() const {
-    for (int i = 0; i < 4; i++) {
-        log::LOGGER.log(log::Logger::LogLevel::DEBUG, "Temp %d: %d\n\r", i, *(this->thermTemps + i));
-    }
-    log::LOGGER.log(log::Logger::LogLevel::DEBUG, "Total Voltage: %d\n\r", totalVoltage);
+void HUDL::updateLCD() {
+    char tempOne[32];
+    std::sprintf(tempOne, "%lu C", *(this->thermTemps + 0));
+
+    char tempTwo[32];
+    std::sprintf(tempTwo, "%lu C", *(this->thermTemps + 1));
+
+    char tempThree[32];
+    std::sprintf(tempThree, "%lu C", *(this->thermTemps + 2));
+
+    char tempFour[32];
+    std::sprintf(tempFour, "%lu C", *(this->thermTemps + 3));
+
+    char voltage[32];
+    std::sprintf(voltage, "%lu v", totalVoltage);
+
+    setTextForSection(0, voltage);
+    //    setTextForSection(2, "25 MPH");
+    //    setTextForSection(2, "3000");
+    setTextForSection(3, tempOne);
+    setTextForSection(4, tempTwo);
+    setTextForSection(5, tempThree);
+    //    setTextForSection(6, "ON");
+    //    setTextForSection(7, "Ready");
+    //    setTextForSection(8, "100 NM");
+}
+
+void HUDL::setTextForSection(uint8_t section, const char* text) {
```

Comment:
If you aren't exposing these functions as public, there's not much reason to even have them private because they're a single line and don't hold any internal logic. Wherever you would use these functions, just make a direct call to the lcd function.